packages/field-plugin/src/createFieldPlugin/createPluginActions/createPluginActions.ts

```diff
@@ -44,7 +44,7 @@ export const createPluginActions: CreatePluginActions = (
   //  isModal and height is not included in the messages to the children and must thus be tracked here.
   //  In future improved versions of the plugin API, this should not be needed.
 
-  const { pushCallback, popCallback } = callbackQueue()
+  const { pushCallback, popCallback } = callbackQueue(10)
```

Comment:
Do we need a limit?

Reply:
I think you're right; it's not needed. I guess if a callback is never received, something is wrong in either Storyfront or in the library, so it shouldn't be possible for the queue size to "explode". 

To limit the size, it's better to remove the promise from the queue immediately when it's popped, rather than letting it wait until the queue size reaches the limit. 

I'll fix it, thanks!